### DISCUSSION TITLE
Testdisk

### DISCUSSION THREAD

**Discussion by 4pastor** [Homebrew/discussions#4249] [Getting started]
### Output of `brew config`

```shell
What is this? I don't know what to put here.
```

### Output of `brew doctor`

```shell
What is this? I don't know what to put here.
```

### Description of issue

Please, what do I do when I install testdisk and see this at the end?

Running `brew cleanup testdisk`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).

Thanks so very much for your help!

**Comment by ZhongRuoyu** [comment]
> What is this? I don't know what to put here.

Run the commands [`brew config`](https://docs.brew.sh/Manpage#config---config) and [`brew doctor`](https://docs.brew.sh/Manpage#doctor-dr---list-checks---audit-debug-diagnostic_check-) and paste the output into the textboxes. The former dumps your Homebrew and system configs, and the latter checks your system for potential issues.

> Please, what do I do when I install testdisk and see this at the end?

That was an automatic cleanup performed after the installation. No action is required from you.

From https://docs.brew.sh/Manpage#cleanup-options-formulacask-: Remove stale lock files and outdated downloads for all formulae and casks, and remove old versions of installed formulae. If arguments are specified, only do this for the given formulae and casks. Removes all downloads more than 120 days old. This can be adjusted with `HOMEBREW_CLEANUP_MAX_AGE_DAYS`.

From https://docs.brew.sh/Manpage#install-options-formulacask-: Unless `HOMEBREW_NO_INSTALL_CLEANUP` is set, brew cleanup will then be run for the installed formulae or, every 30 days, for all formulae.

> Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
> Hide these hints with HOMEBREW_NO_ENV_HINTS (see man brew).

These tell you what to do, if you don't want automatic cleanup to be performed after `brew install`, or don't want to see such hints.

**Comment by 4pastor** [comment]
Thanks so very much for your help! I truly appreciate it.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
